### ISSUE TITLE
`scv.tl.velocity()` fails with `fit_offset=False` and float (upper) percentile `perc`

### ISSUE THREAD

**Issue by JesseRop** [theislab/scvelo#837]
Dear developers,

`scv.tl.velocity(obj, diff_kinetics=True, min_r2=0.1, perc=[7.5, 92.5])`

gives the error below

ValueError: too many values to unpack (expected 2)

No error with

`scv.tl.velocity(obj, diff_kinetics=True, min_r2=0.1, perc=[7.5, 92])`

**Comment by WeilerP** [comment]
@JesseRop, the code snippet is the same in both cases. Also, please provide the package versions you are running this on (`scv.logging.print_versions()`) and ideally a code snippet that I can simply copy and paste to investigate the issue.

**Comment by WeilerP** [comment]
Snippet to reproduce problem:

```python
import scvelo as scv

adata = scv.datasets.pancreas()

scv.pp.filter_and_normalize(adata, min_shared_counts=20, n_top_genes=2000)
scv.pp.moments(adata, n_pcs=30, n_neighbors=30)

scv.tl.velocity(adata, perc=[7.5, 92.5])
```

<details>
<summary> Traceback </summary>

```pytb
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-12-30954f767f43> in <module>
----> 1 scv.tl.velocity(adata, perc=[7.5, 92.5])

~/code/scvelo/scvelo/tools/velocity.py in velocity(data, vkey, mode, fit_offset, fit_offset2, filter_genes, groups, groupby, groups_for_fit, constrain_ratio, use_raw, use_latent_time, perc, min_r2, min_likelihood, r2_adjusted, use_highly_variable, diff_kinetics, copy, **kwargs)
    426                         use_highly_variable=use_highly_variable,
    427                     )
--> 428                 velo.compute_stochastic(fit_offset, fit_offset2, mode, perc=perc)
    429 
    430             write_residuals(adata, vkey, velo._residual, cell_subset)

~/code/scvelo/scvelo/tools/velocity.py in compute_stochastic(self, fit_offset, fit_offset2, mode, perc)
    145             maximum_likelihood(_Ms, _Mu, _Mus, _Mss, fit_offset, fit_offset2)
    146             if mode == "bayes"
--> 147             else leastsq_generalized(
    148                 _Ms,
    149                 _Mu,

~/code/scvelo/scvelo/tools/optimization.py in leastsq_generalized(x, y, x2, y2, res_std, res2_std, fit_offset, fit_offset2, perc)
    129             perc = perc[1]
    130         weights = csr_matrix(
--> 131             get_weight(x, y, perc=perc) | get_weight(x, perc=perc)
    132         ).astype(bool)
    133         x, y = weights.multiply(x).tocsr(), weights.multiply(y).tocsr()

~/code/scvelo/scvelo/tools/optimization.py in get_weight(x, y, perc)
     22         weights = xy_norm >= np.percentile(xy_norm, perc, axis=0)
     23     else:
---> 24         lb, ub = np.percentile(xy_norm, perc, axis=0)
     25         weights = (xy_norm <= lb) | (xy_norm >= ub)
     26     return weights

ValueError: too many values to unpack (expected 2)
```
</details>

Problem is the following check
https://github.com/theislab/scvelo/blob/bc4cdf05fdd1ad0446ad49a06ed517139dc2e365/scvelo/tools/optimization.py#L18

**Comment by JesseRop** [comment]
Many thanks for your prompt attention to this. My apologies for not including a reproducible snippet and thanks for including one. Kindly let me know once it is resolved.

**Comment by WeilerP** [comment]
Fixed by #839.
